.github/scripts/install-dependencies.sh

```diff
@@ -4,7 +4,24 @@ source `dirname ${BASH_SOURCE[0]}`/config.sh
 
 echo "::group::Install Dependencies"
     sudo apt update
-    sudo apt install -y build-essential autogen autoconf automake autotools-dev binutils-for-build texinfo bison flex ccache docbook2x xmlto zlib1g-dev libc6-dev-arm64-cross libc6-dev-amd64-cross dejagnu
+    sudo apt install -y \
+      build-essential \
+      autogen \
+      autoconf \
+      automake \
+      autotools-dev \
+      binutils-for-build \
+      texinfo \
+      bison \
+      flex \
+      ccache \
+      docbook2x \
+      xmlto \
+      zlib1g-dev \
+      libc6-dev-arm64-cross \
+      libc6-dev-amd64-cross \
+      dejagnu \
+      zstd
```

Comment:
what is the main purpose of using zstd here compare to standard packing?

Reply:
`zstd` is the default packing for `actions/cache` if it's available and the only one that works accross different runner OSes. If not, it falls back to what is available.

Reply:
Reordered alphabetically.